git-sync/main.go

```diff
@@ -66,8 +84,24 @@ func main() {
 	if _, err := exec.LookPath("git"); err != nil {
 		log.Fatalf("required git executable not found: %v", err)
 	}
+
+	if *flUsername != "" && *flPassword != "" {
```

Comment:
I'm curious.  Why pointers?  is `len(flUsername)==0` or `flUsername==""` insufficient?